docs/reference/tab-widgets/inference-api/infer-api-mapping.asciidoc

```diff
@@ -99,4 +99,42 @@ In this example, the name of the field is `content`. It must be referenced in
 the {infer} pipeline configuration in the next step.
 <6> The field type which is text in this example.
 
-// end::openai[]
\ No newline at end of file
+// end::openai[]
+
+// tag::azure-openai[]
+
+[source,console]
+--------------------------------------------------
+PUT azure-openai-embeddings
+{
+  "mappings": {
+    "properties": {
+      "content_embedding": { <1>
+        "type": "dense_vector", <2>
+        "dims": 1536, <3>
+        "element_type": "float",
+        "similarity": "dot_product" <4>
+      },
+      "content": { <5>
+        "type": "text" <6>
+      }
+    }
+  }
+}
+--------------------------------------------------
+<1> The name of the field to contain the generated tokens. It must be referenced
+in the {infer} pipeline configuration in the next step.
+<2> The field to contain the tokens is a `dense_vector` field.
+<3> The output dimensions of the model. Find this value in the
+https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#embeddings-models[Azure OpenAI documentation]
+of the model you use.
+<4> The faster` dot_product` function can be used to calculate similarity
```

Comment:
Is it okay to keep the OpenAI reference and the docs link in this step or should it be changed to something Azure OpenAI specific?

Reply:
Are you referring to the link above (line 129)? This is the Azure-specific documentation... unless I'm missing something.